### DISCUSSION TITLE
Unify Zen's and Firefox's sidebars

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#809] [Ideas]
### What feature would you like to see?

I would like to see Firefox's and Zen's sidebar merged into one.

### What platform are you seeing the problem on?

Linux

### Additional context

![image](https://github.com/user-attachments/assets/6e76d8fc-92cb-401c-9ba2-a423d992ee1e)

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/523_

**Comment by SomeDudeBrosky** [comment]
This seems more like a visual bug than a feature request
